lib/internal/main/worker_thread.js

```diff
@@ -135,21 +138,23 @@ port.on('message', (message) => {
 
     require('internal/worker').assignEnvironmentData(environmentData);
 
-    // The counter is only passed to the workers created by the main thread, not
-    // to workers created by other workers.
-    let cachedCwd = '';
-    let lastCounter = -1;
-    const originalCwd = process.cwd;
-
-    process.cwd = function() {
-      const currentCounter = Atomics.load(cwdCounter, 0);
-      if (currentCounter === lastCounter)
+    if (typeof SharedArrayBuffer !== 'undefined') {
```

Comment:
`Atomics` can be disabled using `--no-harmony-atomics`
```suggestion
    if (SharedArrayBuffer !== undefined && Atomics !== undefined) {
```

Reply:
We do not support V8 flags. As such, I would not want to special handle this.

Reply:
We have to use typeof though: checking for a non existing variable would otherwise end up in a ReferenceError.

Reply:
I don't think we do, because we're using `const {Atomics, SharedArrayBuffer} = globalThis` at the top of the file.